vhdbuilder/scripts/automate_release.sh

```diff
@@ -32,7 +32,7 @@ trigger_ev2_artifacts() {
 
 create_release() {
     echo "creating SIG release for VHD with build ID: $VHD_BUILD_ID"
-    API_ENDPOINT="https://${ADO_ORG}.vsrm.visualstudio.com/${ADO_PROJECT}/_apis/Release/releases?api-version=5.1"
+    API_ENDPOINT="https://msazure.vsrm.visualstudio.com/CloudNativeCompute/_apis/Release/releases?api-version=5.1"
```

Comment:
I think the idea was not to make those public ?

Reply:
it's already all over the place lol, though I can get rid of them from here if we want